### PR TITLE
[KIECLOUD-505] 7.12.x Update https.sh script to use the one provided by wildfly-cekit-modules

### PR DESCRIPTION
**Pull request by desmax74**
Signed-off-by: desmax74 <mdessi@redhat.com>
(cherry picked from commit ee73bec905233e292b2c102402b2fe4a77dae262)
See:https://issues.redhat.com/browse/KIECLOUD-505

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: businesscentral-monitoring/image.yaml [hunk 1]
```diff
@@ -119,6 +119,8 @@ modules:
       version: "1.0-kie"
     - name: os-eap7-launch
       version: "1.0-kie"
+    - name: jboss.container.wildfly.launch.access-log-valve
```
Comment (spolti) [inline businesscentral-monitoring/image.yaml:122]: same here

Comment (spolti) [inline businesscentral-monitoring/image.yaml:122]: same here

**Review by spolti — CHANGES_REQUESTED** [review]

File: businesscentral-monitoring/image.yaml (hunk 1 above)
Comment (desmax74) [reply businesscentral-monitoring/image.yaml]: Fixed

**Review by spolti — DISMISSED** [review]

**Review by amadhusu — APPROVED** [review]

**Comment by spolti** [comment]
@desmax74 can you please rebase?

**Review by spolti — APPROVED** [review]

**Pull request merged**